### DISCUSSION TITLE
On Screen viewing?

### DISCUSSION THREAD

**Discussion by snowite0** [uxbox/uxbox#581] [Q&A] [answered]
So, I am on a desktop pc.  I can see all the screens VERY LITTLE. So little in fact, that I can't edit them.  Or, have to guess what I am editing because I cannot zoom the workspace larger or smaller. Using my mouse wheel does nothing and there is nothing for me to click to make the screens bigger.  see image.
![image](https://user-images.githubusercontent.com/30342297/107045154-31e6e100-6793-11eb-9340-54fb6387ccaf.png)

**Comment by elhombretecla** [comment, marked as answer]
Hi!!! You can zoom over the canvas with ctrl + mouse wheel or using the zoom controllers at the top right corner of the workspace :)
<img width="358" alt="zoom-controllers" src="https://user-images.githubusercontent.com/1045247/107492021-31cc5400-6b8c-11eb-87f8-2fa236c9f2f8.png">

**Comment by snowite0** [comment]
thank you!
